server/datastore/mysql/migrations/tables/20220526123327_RenameCVEScoresToCVEMeta.go

```diff
@@ -0,0 +1,35 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220526123327, Down_20220526123327)
+}
+
+func Up_20220526123327(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+RENAME TABLE
+    cve_scores TO cve_meta
+`)
+	if err != nil {
+		return errors.Wrapf(err, "rename table")
+	}
+
+	_, err = tx.Exec(`
+ALTER TABLE cve_meta
+    ADD published TIMESTAMP NULL DEFAULT NULL
```

Comment:
We might want to add an index to `published`

Reply:
I'll let you add an index if this is necessary. I don't like to add indexes unless it actually improves performance. Also, this table will be quite small.

Reply:
Sounds good